.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Build'
+on:
+  - 'push'
+jobs:
+  release:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: 'actions/checkout@v3'
+      - name: 'Setup .NET Core SDK'
+        uses: 'actions/setup-dotnet@v3'
+        with:
+          dotnet-version: '6.0.x'
```

Comment:
We need .net standard 2.1 for use within Unity3d. Don't 6.0 can't be included unfortunately: https://docs.unity3d.com/Manual/dotnetProfileSupport.html

Reply:
Ah it looks like I was confused by a version mismatch between `PxCs` and `PxCs.Tests`, since the Tests use .NET 6.0.

Reply:
Okay so I did some more digging and I ended up looking at the strings in `PxCs.dll`. It says `.NET Standard 2.1`:

![bilde](https://user-images.githubusercontent.com/20187490/231187134-f44d2b23-223d-4972-b83e-98b09d1bf720.png)

So I believe that this version refers to the `.NET Core SDK` which does support `.NET Standard 2.1` as per https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1#net-standard-versions:

![bilde](https://user-images.githubusercontent.com/20187490/231187655-44ef12f1-5583-44e0-a87c-1900a3211d4d.png)

and it's the solution file that ends up selecting the actual framework to use.

Reply:
I think you're absolutely right! On my local machine I also have .NET 7 installed and build as .NET 6 (Tests) and .NET Standard 2.1 (PxCs).

I will approve and check the built DLL if working.